scripts/combine/setupCombineWMass.py

```diff
@@ -20,7 +20,7 @@ def make_parser(parser=None):
         parser = common.common_parser_combine()
     parser.add_argument("--fitvar", nargs="+", help="Variable to fit", default=["pt", "eta"])
     parser.add_argument("--noEfficiencyUnc", action='store_true', help="Skip efficiency uncertainty (useful for tests, because it's slow). Equivalent to --excludeNuisances '.*effSystTnP|.*effStatTnP' ")
-    parser.add_argument("--ewUnc", action='store_true', help="Include EW uncertainty")
+    parser.add_argument("--ewUnc", type=str, nargs="*", default=[], choices=["horacenloew", "winhacnloew"], help="Include EW uncertainty")
```

Comment:
Should probably make the default winhacnloew?

Reply:
Not sure, we then also would have to change the histmaker default

Reply:
From what you showed yesterday that seems well motivated, right? is the Horace correction currently applied by default?

Reply:
No, we don't apply any by default.

Reply:
I think we should NOT apply any correction by default (it is not fully differential, there are some approximations in the reweighting etc).  But clearly at the moment winhac should be the default for the uncertainty for the W case.

Reply:
> I think we should NOT apply any correction by default (it is not fully differential, there are some approximations in the reweighting etc). But clearly at the moment winhac should be the default for the uncertainty for the W case.

Ok sorry by "apply" I meant, we don't assign any uncertainty at the moment